app/src/main/java/org/mifos/mobilebanking/ui/fragments/LoanApplicationFragment.java

```diff
@@ -95,15 +95,17 @@
     private ArrayAdapter<String> loanPurposeAdapter;
 
     private LoanTemplate loanTemplate;
-    private DialogFragment mfDatePicker;
+    private DialogFragment mfDatePickerDisbursement = MFDatePicker.newInstance(
+            this, MFDatePicker.STARTS_CURRENT_DAY);
+    private DialogFragment mfDatePickerSubmission = MFDatePicker.newInstance(
```

Comment:
If you are initialising these in the methods then why are initialising here. Also, we would need any one `DialogFragment` as they are then initialised in their respective methods.